python/pyspark/sql/classic/dataframe.py

```diff
@@ -1795,7 +1791,9 @@ def executionInfo(self) -> Optional["ExecutionInfo"]:
         )
 
     @property
-    def plot(self) -> PySparkPlotAccessor:
+    def plot(self) -> "PySparkPlotAccessor":
```

Comment:
nit: we may need `# type: ignore[name-defined] # noqa: F821` :)

Reply:
Good catch! Thanks :)